src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -72,12 +72,18 @@ public TestParameters WithFixProviderData(object fixProviderData)
 
         protected abstract TestWorkspace CreateWorkspaceFromFile(string initialMarkup, TestParameters parameters);
 
+        private TestParameters WithRegularOptions(TestParameters parameters)
+            => parameters.WithParseOptions(parameters.parseOptions?.WithKind(SourceCodeKind.Regular));
```

Comment:
this is an important change so that TestMissingInRegularAndScriptAsync and the other tests actually use the language version we asked for by giving ParseOptions inside TestParameters

Reply:
did this reveal any errant tests that were passing when tehy shouldn't hve been?

Reply:
actually no, i was surprised by that

Reply:
but it revealed a lot of tests that would have passed even if the code they were supposed to test didn't work